--- wv-0.7.0/configure.in~	Mon Oct 29 17:43:07 2001
+++ wv-0.7.0/configure.in	Thu Nov  8 04:46:09 2001
@@ -1,11 +1,8 @@
 dnl Process this file with autoconf to produce a configure script.
 AC_INIT(fib.c)
-AC_CONFIG_HEADER(config.h)
 
-dnl I did this wrong, I think. I should have used AC_DEFINE(VERSION,"$VERSION") or similar
-dnl but what's done is done... (fjf)
-VERSION=0.7.0
-AC_SUBST(VERSION)
+AM_INIT_AUTOMAKE(wv, 0.7.0)
+AM_CONFIG_HEADER(config.h)
 
 # Tests for Windows
 AC_CYGWIN
@@ -22,13 +19,6 @@
 dnl AC_PROG_CC_NO_G
 AC_PROG_CC
 
-dnl This is really a check for Mac OS X.1 (or darwin 1.4, whatever...)
-suppress_md=`cat /usr/share/man/man1/ld.1 2>&1 | grep multiply_defined`
-if test "x$suppress_md" != "x"; then
-  CFLAGS="$CFLAGS -Wno-long-long"
-  LDFLAGS="$LDFLAGS -Wl,-multiply_defined -Wl,suppress"
-fi
-
 AC_PROG_MAKE_SET
 AC_PROG_INSTALL
 AC_PROG_RANLIB
