diff -Nur wv-0.6.5.orig/Makefile.in wv-0.6.5/Makefile.in
--- wv-0.6.5.orig/Makefile.in	Tue May 22 15:12:16 2001
+++ wv-0.6.5/Makefile.in	Tue May 22 16:02:07 2001
@@ -82,7 +82,7 @@
 
 VPATH = @srcdir@
 
-CFLAGS = @CPPFLAGS@ @CFLAGS@ @DEFS@ @ZLIB_CFLAGS@ @GLIB_CFLAGS@ -ansi -pedantic -I. -I./ -I$(srcdir)/ -I$(srcdir)/magick/ -I$(srcdir)/expat/xmlparse/ -DXMLCONFIG=\"$(datadir)/wv/wvConfig.xml\" -DHTMLCONFIG=\"$(datadir)/wv/wvHtml.xml\" -I$(srcdir)/libole2/ -I$(srcdir)/oledecod/ @WMF_CFLAGS@ @ICONV_CFLAGS@
+CFLAGS = @CPPFLAGS@ @CFLAGS@ @DEFS@ @ZLIB_CFLAGS@ @GLIB_CFLAGS@ -ansi -pedantic -I. -I./ -I$(srcdir)/ @MAGICK_CFLAGS@ -I$(srcdir)/expat/xmlparse/ -DXMLCONFIG=\"$(datadir)/wv/wvConfig.xml\" -DHTMLCONFIG=\"$(datadir)/wv/wvHtml.xml\" -I$(srcdir)/libole2/ -I$(srcdir)/oledecod/ @WMF_CFLAGS@ @ICONV_CFLAGS@
 
 ifneq ($(HAVE_GLIB),1)
 CFLAGS += "-I$(srcdir)/glib-wv/"
diff -Nur wv-0.6.5.orig/bmptopng.c wv-0.6.5/bmptopng.c
--- wv-0.6.5.orig/bmptopng.c	Tue Apr 10 18:51:24 2001
+++ wv-0.6.5/bmptopng.c	Tue May 22 16:02:59 2001
@@ -1,4 +1,4 @@
-#include "magick/magick.h"
+#include <magick/magick.h>
 #include <string.h>
 #ifdef HAVE_CONFIG_H
 #include "config.h"
@@ -23,7 +23,7 @@
     GetImageInfo (&image_info);
     sprintf (buffer, "%s.bmp", prefix);
     strcpy (image_info.filename, buffer);
-    image = ReadBMPImage (&image_info);
+    image = ReadImage (&image_info);
     if (image == (Image *) NULL)
 	return (1);
     sprintf (buffer, "%s.png", prefix);
@@ -31,7 +31,7 @@
     strcpy (image_info.filename, buffer);
     SetImageInfo (&image_info, 1);
     strcpy (image->filename, buffer);
-    WritePNGImage (&image_info, image);
+    WriteImage (&image_info, image);
 
     DestroyImage (image);
     return (0);
--- wv-0.6.7/configure.in.orig	Sat Sep  1 11:05:44 2001
+++ wv-0.6.7/configure.in	Sat Sep  1 11:15:31 2001
@@ -376,9 +376,9 @@
     AC_CHECK_HEADER(magick/magick.h,
         AC_CHECK_LIB(Magick, GetImageInfo,[
             MAGICK_CFLAGS=$m_cppflags
-            MAGICK_LIBS=$m_ldflags
-            MAGICK="BMAGICK"
-            MAGICKOBJS="\$(RMAGICKOBJS)"]))
+            MAGICK_LIBS="$m_ldflags -lMagick"
+            MAGICK=" "
+            MAGICKOBJS=""]))
 fi
 
 if test -z "$MAGICK"; then
