--- wv-1.0.3.orig/GNUmakefile.am	Tue Jan 15 10:58:17 2002
+++ wv-1.0.3/GNUmakefile.am	Thu Feb 21 22:55:58 2002
@@ -26,20 +26,30 @@
 LIBEXPORTER = 
 endif
 
+if BUILD_MAGICK
+DIRMAGICK = magick
+INCMAGICK = -I$(srcdir)
+LIBMAGICK = magick/libmagick.la
+else
+DIRMAGICK =
+INCMAGICK =
+LIBMAGICK =
+endif
+
 DIST_SUBDIRS = xml help wingdingfont patterns glib-wv libole2 magick expat exporter .
 
-SUBDIRS = xml help wingdingfont patterns $(DIRGLIB) libole2 magick $(DIREXPAT) $(DIREXPORTER) .
+SUBDIRS = xml help wingdingfont patterns $(DIRGLIB) libole2 $(DIRMAGICK) $(DIREXPAT) $(DIREXPORTER) .
 
 DEFS = @DEFS@ -DVERSION=\"@VERSION@\" -DXMLCONFIG=\"$(pkgdatadir)/wvConfig.xml\" -DHTMLCONFIG=\"$(pkgdatadir)/wvHtml.xml\" -DWVDATADIR=\"$(datadir)/wv\"
 
 INCLUDES = -I. -I$(top_builddir) -I$(srcdir) \
-	-I$(srcdir)/magick -I$(srcdir)/libole2 $(INCGLIB) $(INCEXPAT)
+	$(INCMAGICK) -I$(srcdir)/libole2 $(INCGLIB) $(INCEXPAT)
 
 CFLAGS = @CFLAGS@ @ANSI_CFLAGS@
 
-CPPFLAGS = @CPPFLAGS@ @WMF_CFLAGS@ @XML_CFLAGS@ @GLIB_CFLAGS@ @ICONV_CFLAGS@ @PNG_CFLAGS@ @ZLIB_CFLAGS@
+CPPFLAGS = @CPPFLAGS@ @WMF_CFLAGS@ @XML_CFLAGS@ @GLIB_CFLAGS@ @ICONV_CFLAGS@ @PNG_CFLAGS@ @ZLIB_CFLAGS@ @MAGICK_CFLAGS@
 
-WVLIBS = @WMF_LIBS@ @XML_LIBS@ @GLIB_LIBS@ @ICONV_LIBS@ @PNG_LIBS@ @ZLIB_LIBS@ @LIBS@ -lm
+WVLIBS = @WMF_LIBS@ @XML_LIBS@ @GLIB_LIBS@ @ICONV_LIBS@ @PNG_LIBS@ @ZLIB_LIBS@ @MAGICK_LIBS@ @LIBS@ -lm
 
 include_HEADERS = wv.h
 
@@ -57,7 +67,7 @@
 wvConvert_LDADD = libwv.la $(GETOPT)
 wvVersion_LDADD = libwv.la $(GETOPT)
 
-libwv_la_LIBADD = magick/libmagick.la libole2/libole2.la $(LIBGLIB) $(LIBEXPAT) $(LIBEXPORTER) $(WVLIBS)
+libwv_la_LIBADD = $(LIBMAGICK) libole2/libole2.la $(LIBGLIB) $(LIBEXPAT) $(LIBEXPORTER) $(WVLIBS)
 libwv_la_LDFLAGS = -version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) -release $(LT_RELEASE) -export-dynamic
 libwv_la_SOURCES = \
 	support.c \
diff -Nur wv-0.7.1.orig/bmptopng.c wv-0.7.1/bmptopng.c
--- wv-0.7.1.orig/bmptopng.c	Tue Apr 10 18:36:48 2001
+++ wv-0.7.1/bmptopng.c	Thu Feb 21 22:56:31 2002
@@ -1,4 +1,7 @@
-#include "magick/magick.h"
+#include <stdio.h>
+#include <time.h>
+#include <sys/types.h>
+#include <magick/api.h>
 #include <string.h>
 #ifdef HAVE_CONFIG_H
 #include "config.h"
@@ -23,15 +23,27 @@
     GetImageInfo (&image_info);
     sprintf (buffer, "%s.bmp", prefix);
     strcpy (image_info.filename, buffer);
+#ifdef HAVE_MAGICK
+    image = ReadImage (&image_info, NULL);
+#else
     image = ReadBMPImage (&image_info);
+#endif
     if (image == (Image *) NULL)
 	return (1);
     sprintf (buffer, "%s.png", prefix);
 
     strcpy (image_info.filename, buffer);
+#ifdef HAVE_MAGICK
+    SetImageInfo (&image_info, 1, NULL);
+#else
     SetImageInfo (&image_info, 1);
+#endif
     strcpy (image->filename, buffer);
+#ifdef HAVE_MAGICK
+    WriteImage (&image_info, image);
+#else
     WritePNGImage (&image_info, image);
+#endif
 
     DestroyImage (image);
     return (0);
diff -Nur wv-0.7.1.orig/configure.ac wv-0.7.1/configure.ac
--- wv-0.7.1.orig/configure.ac	Wed Jan 16 11:39:46 2002
+++ wv-0.7.1/configure.ac	Thu Feb 21 23:02:39 2002
@@ -215,6 +215,52 @@
 CPPFLAGS=$_cppflags
 LDFLAGS=$_ldflags
 
+dnl system Magick support
+dnl ========================================================
+
+_cppflags=$CPPFLAGS
+_ldflags=$LDFLAGS
+
+AC_ARG_WITH(Magick,[  --with-Magick[=DIR]    use Magick library in DIR],[
+	if test "x$withval" = "xyes"; then
+		MAGICK_DIR=""
+	elif test "x$withval" != "xno"; then
+		MAGICK_DIR=$withval
+		CPPFLAGS="$CPPFLAGS -I$withval/include"
+		LDFLAGS="$LDFLAGS -L$withval/lib"
+	fi
+],[	MAGICK_DIR=""
+])
+
+AC_CHECK_HEADER(magick/magick.h,[
+	AC_CHECK_LIB(Magick, ReadImage,[
+		AC_DEFINE(HAVE_MAGICK,1,[define if you have libMagick])
+		HAVE_MAGICK=1
+		if test "x$MAGICK_DIR" != "x"; then
+			MAGICK_CFLAGS="-I$MAGICK_DIR/include"
+			MAGICK_LIBS="-L$MAGICK_DIR/lib -lMagick"
+		else
+			MAGICK_CFLAGS=""
+			MAGICK_LIBS="-lMagick"
+		fi
+	],[	HAVE_MAGICK=0
+		MAGICK_CFLAGS=""
+		MAGICK_LIBS=""
+	])
+],[	HAVE_MAGICK=0
+	MAGICK_CFLAGS=""
+	MAGICK_LIBS=""
+])
+
+AC_SUBST(HAVE_MAGICK)
+AM_CONDITIONAL(BUILD_MAGICK,[ test "$HAVE_MAGICK" = "0" ])
+
+AC_SUBST(MAGICK_CFLAGS)
+AC_SUBST(MAGICK_LIBS)
+
+CPPFLAGS=$_cppflags
+LDFLAGS=$_ldflags
+
 dnl system XML support
 dnl ========================================================
 
